<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/infant/indoor_dkan/2019-04-04-14-27-47.bag"/>
	
	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_store)rviz_config/lower_pc_hz.rviz"/>

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- main -->
	<node pkg="pc_store" name="lower_pc_hz" type="lower_pc_hz" output="screen">
		<remap from="/cloud" to="/velodyne_points"/>
		<remap from="/cloud/lower_hz" to="/velodyne_points/lower_hz"/>
		<param name="target_hz" type="double" value="1"/>
	</node>
</launch>
